.github/workflows/assets.yml

```diff
@@ -75,6 +75,13 @@ jobs:
              mv assets/kernel assets/kernel.gz
              gzip -d assets/kernel.gz
           fi
+      - name: Build SBOM and collected sources
+        run: |
+            git clone ${{ github.event.repository.html_url }} .
+            git reset --hard ${{ github.event.workflow_run.head_branch }}
+            make sbom collected_sources
+            mkdir -p assets
+            cp -rf collected_sources.tar.gz rootfs.spdx.json assets
```

Comment:
`cp` or `mv`? 

Also, doesn't it put the assets in `dist/...` and not in the local directory?

Reply:
It should be`mv`. 

Yes, but the assets are inside the dist/amd64(assuming the amd64 arch) folder. That variable is defined inside the Makefile. I don't want to hardcode it to the specific folder. Would it make sense to generate the artifacts in the current folder or I can also use a shell command to copy these to the assets folder.

Reply:
You are right, the `mv` should be defined here in the workflow file, similar to the other steps.

So is the issue knowing which directory they are in? The current definition above just assumes they are in the current directory, which `make sbom collected_sources` does not do?

Reply:
When we execute the make command the files are created as per the below directory structure. Since we have a platform dependent path which essentially is not the case for our assets. I will use a simple find command to find the assets and move it to the assets folder. 


`bash-3.2$ tree -L 4
.
└── amd64
    └── 0.0.0-master-0517cd29-dirty-2023-04-26.13.50
        ├── installer
        │   ├── eve_version
        │   ├── grub.cfg
        │   ├── ipxe.efi.cfg
        │   └── rootfs.spdx.json
        └── rootfs.tar

4 directories, 5 files
`

Reply:
Yeah, that should work. Mind, I would be concerned about `find` finding multiple, but it shouldn't. You can check that it returns just 1 before moving it.

Also, isn't there a `current` symlink also created?

Reply:
I was looking for something like this but I don't think there is any such symlink created. I will use the find command. Ideally there should be only one because make command would be executed only once per workflow. To be on safe side I will do `rm` on the dist directory.

Reply:
Added the info targets to get the path of the artefacts created